### PR TITLE
Add some basic usage information about qgis_process

### PR DESCRIPTION
**Pull request by m-kuhn**
Closes https://github.com/qgis/QGIS-Documentation/issues/6192

### CODE REVIEW

File: docs/user_manual/processing/standalone.rst [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+Using processing from the command line
+======================================
+
+.. only:: html
+
+   .. contents::
+      :local:
+
+QGIS comes with a tool called `qgis_process` which allows you to call
+processing algorithms directly from the command line without starting
```
Comment (nyalldawson) [inline docs/user_manual/processing/standalone.rst:12]:
```suggestion
processing algorithms and models directly from the command line without starting
```

**Comment by nyalldawson** [comment]
Looks good!

File: docs/user_manual/processing/standalone.rst [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+Using processing from the command line
+======================================
+
+.. only:: html
+
+   .. contents::
+      :local:
+
+QGIS comes with a tool called `qgis_process` which allows you to call
```
Comment (DelazJ) [inline docs/user_manual/processing/standalone.rst:11]: Afaict, in html, \` is not rendered with a special formatting
```suggestion
QGIS comes with a tool called ``qgis_process`` which allows you to call
```

File: docs/user_manual/processing/standalone.rst [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+processing algorithms and models directly from the command line without starting
+QGIS desktop itself.
+
+The command `list` can be used to get a list of all available providers
```
Comment (DelazJ) [inline docs/user_manual/processing/standalone.rst:15]:
```suggestion
The command ``list`` can be used to get a list of all available providers
```

File: docs/user_manual/processing/standalone.rst [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+and algorithms.
+
+.. code-block:: bash
```
Comment (DelazJ) [inline docs/user_manual/processing/standalone.rst:18]: Need a blank line if I'm not wrong
```suggestion
.. code-block:: bash

```

File: docs/user_manual/processing/standalone.rst [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+   qgis_process list
+
+The command `run` can be used to run an algorithm or model.
```
Comment (DelazJ) [inline docs/user_manual/processing/standalone.rst:21]:
```suggestion
The command ``run`` can be used to run an algorithm or model.
```

File: docs/user_manual/processing/standalone.rst [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+Specify the name of the algorithm or a path to a model as first parameter.
+
+.. code-block:: bash
```
Comment (DelazJ) [inline docs/user_manual/processing/standalone.rst:24]:
```suggestion
.. code-block:: bash

```

File: docs/user_manual/processing/standalone.rst [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+   qgis_process run qgis:buffer -- INPUT=source.shp DISTANCE=2 OUTPUT=buffered.shp
+
+The `run` command also supports further parameters.
```
Comment (DelazJ) [inline docs/user_manual/processing/standalone.rst:27]:
```suggestion
The ``run`` command also supports further parameters.
```

File: docs/user_manual/processing/standalone.rst [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+
+ - `--json` will format stdout output in a JSON structured way.
+ - `--ellipsoid` will set the ellipsoid to the specified one.
+ - `--distance_units` will use the specified distance units.
+ - `--area_units` will use the specified area units.
+ - `--project_path` will load the specified project for running the algorithm.
```
Comment (DelazJ) [inline docs/user_manual/processing/standalone.rst:33]:
```suggestion
 - `--json` will format stdout output in a JSON structured way.
 - ``--ellipsoid`` will set the ellipsoid to the specified one.
 - ``--distance_units`` will use the specified distance units.
 - ``--area_units`` will use the specified area units.
 - ``--project_path`` will load the specified project for running the algorithm.
```

File: docs/user_manual/processing/standalone.rst [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+
+The command `help` can be used to get further information about commands or algorithms
```
Comment (DelazJ) [inline docs/user_manual/processing/standalone.rst:35]:
```suggestion
The command ``help`` can be used to get further information about commands or algorithms.
```

File: docs/user_manual/processing/standalone.rst [hunk 10]
```diff
@@ -0,0 +1,38 @@
...
+
+.. code-block:: bash
```
Comment (DelazJ) [inline docs/user_manual/processing/standalone.rst:37]:
```suggestion
.. code-block:: bash

```

**Review by DelazJ — APPROVED** [review]
Some formatting suggestions/fixes
Thanks @m-kuhn

**Comment by m-kuhn** [comment]
Thx @nyalldawson and @DelazJ 

Let me know if there is more

**Comment by DelazJ** [comment]
Maybe link the issues addressed?

**Comment by m-kuhn** [comment]
done

**Pull request merged**